packages/common/src/compiler/filtersCompiler.ts

```diff
@@ -301,6 +301,10 @@ export const renderFilterRuleSql = (
     startOfWeek: WeekDay | null | undefined,
     adapterType: SupportedDbtAdapter,
 ): string => {
+    if ('disabled' in filterRule && filterRule.disabled) {
```

Comment:
```suggestion
    if (!!filterRule.disabled) {
```

Reply:
thank you 🙏 
Removed `!!` as ESLint told me it's redundant!